recipes/slda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "slda" %}
+{% set version = "0.1.5" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/Savvysherpa/slda/archive/{{ version }}.tar.gz
+    sha256: 22854daaa3117587a85f514d891d7917ed881bd68d887dd9ac1fab9aede0c78a
+
+build:
+    number: 0
+    skip: True  # [win or not py3k]
+
+requirements:
+    build:
+        - cython
+        - cythongsl
+        - gsl
```

Comment:
`cythongsl` depends on `gsl`. To avoid over-specifying the dependencies you can remove `gsl` from both `build` and `run` sections.